src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -230,6 +230,30 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
         }
 
+        /// <summary>
+        /// Gets an &lt;input&gt; element's "type" attribute value based on the given <paramref name="modelExplorer"/>
+        /// or <see cref="InputType"/>.
+        /// </summary>
+        /// <param name="modelExplorer">The <see cref="ModelExplorer"/> to use.</param>
+        /// <param name="inputTypeHint">When this method returns, contains a type hint associated with the given
+        /// <paramref name="modelExplorer"/>.</param>
```

Comment:
Closer to `... contains the string, often the name of a <see cref="ModelMetadata.ModelType"/> base class, used to determine this method's return value.`